bucket/obs-studio27.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "27.2.4",
+    "description": "Video recording and live streaming software",
+    "homepage": "https://obsproject.com",
+    "license": "GPL-2.0-only",
+    "notes": [
+        "Add Virtual Camera module by running: \"$dir\\data\\obs-plugins\\win-dshow\\virtualcam-install.bat\"",
+        "Remove it by running: \"$dir\\data\\obs-plugins\\win-dshow\\virtualcam-uninstall.bat\""
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://cdn-fastly.obsproject.com/downloads/OBS-Studio-27.2.4-Full-x64.zip",
+            "hash": "fae9840096c43a62b410d91985b4b5fdd5b0d6e8d54ef4afe7af47180da8ae21",
+            "shortcuts": [
+                [
+                    "bin\\64bit\\obs64.exe",
+                    "OBS Studio"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://cdn-fastly.obsproject.com/downloads/OBS-Studio-27.2.4-Full-x86.zip",
+            "hash": "51bacea13391874f2902575d2d8ab6e6eb6bd7c90682c7461e7cde15dec99c22",
+            "shortcuts": [
+                [
+                    "bin\\32bit\\obs32.exe",
+                    "OBS Studio"
+                ]
+            ]
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\portable_mode.txt\")) { New-Item \"$dir\\portable_mode.txt\" | Out-Null }",
+    "persist": [
+        "config",
+        "portable_mode.txt"
+    ],
+    "checkver": {
+        "url": "https://github.com/obsproject/obs-studio/releases",
+        "regex": "OBS-Studio-([\\d.]+)-Full-x86\\.zip"
```

Comment:
Should we disable checkver & autoupdate entirely? It seems unlikely there will be any updates to the 27.x branch

Reply:
I changed the regex to `x86` in the off chance they publish again. Imo, removing x86 builds isn't helpful as many emulation layers use it, such as arm.